features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/SessionReplayFeature.kt

```diff
@@ -40,7 +41,7 @@ internal class SessionReplayFeature(
     customEndpointUrl: String?,
     internal val privacy: SessionReplayPrivacy,
     private val rateBasedSampler: Sampler,
-    private val sessionReplayRecorderProvider: (RecordWriter, Application) -> Recorder
+    private val sessionReplayRecorderProvider: (ResourcesFeature, RecordWriter, Application) -> Recorder
```

Comment:
In fact you don't need the whole `ResourcesFeature` in the recorder, you need only `writer` part of it.